app/services/streamlabels/definitions.ts

```diff
@@ -0,0 +1,861 @@
+import { TPlatform } from 'services/platforms';
```

Comment:
Most of this file is copied from the existing streamlabels app.  I munged the data a bit, removed mixer definitions for now, and defined some interfaces based on the structure of the data.